pkg/config/jobs.go

```diff
@@ -282,6 +295,8 @@ type Periodic struct {
 	// Tags for config entries
 	Tags []string `json:"tags,omitempty"`
 
+	RetriggerFailed *RetriggerFailed `json:"retrigger_failed_run,omitempty"`
```

Comment:
nit: perhaps something like `retry_on_failure` will be a better name

Reply:
I would like to keep it as it was already discussed during previous review and changed upon request. Also it's closer to what was requested in the issue.